### ISSUE TITLE
Electron 6:  'setHighlightMode function' is deprecated and will be removed.

### ISSUE THREAD

**Issue by amaurymartiny** [maxogden/menubar#243]
[//]: # "Hey there and thank you for using Menubar's issue tracker!"
[//]: # "Please fill in these information to describe your issue."

### Description

Seeing this warning logged

```
(electron) 'setHighlightMode function' is deprecated and will be removed.
```

### Steps to Reproduce the Problem

1. run the hello-world example

### Expected Behaviour

No warnings

### Actual Behaviour

warnings

### Specifications

- Menubar version: 6.0.6
- Platform: macOs
- Electron version: 6.0.0

**Comment by e3duardo** [comment]
According to the documentation, this function is deprecated and will be removed in version 7.

https://electronjs.org/docs/api/tray#traysethighlightmodemode-macos

https://electronjs.org/docs/api/breaking-changes#tray

**Comment by sem4phor** [comment]
What does hacktoberfest mean? today electron 7 will be released where this function will be removed. :O

**Comment by amaurymartiny** [comment]
Hacktoberfest means https://hacktoberfest.digitalocean.com/ - to encourage people to contribute to open-source.

If no-one is volunteering, I'll take care of it.

**Comment by amaurymartiny** [comment]
Fixed in [`menubar@7.0.0`](https://www.npmjs.com/package/menubar/v/7.0.0) 🚀
